plugins/woocommerce/tests/e2e-pw/global-setup.js

```diff
@@ -1,15 +1,6 @@
 const { chromium, expect } = require( '@playwright/test' );
+const { admin, customer } = require( './test-data/data' );
```

Comment:
Instead of using a lot of environment variables like `ADMIN_USER`, `ADMIN_PASSWORD`, etc., I added admin and customer objects in `test-data/data.js` that tests can use to retrieve their username, password, billing, and other details.